wwwroot/buy-anything.html

```diff
@@ -0,0 +1,414 @@
+                        <div class="col-md-5">
+                            <div class="sticky-top">
+                                <h2 class="pt-header title-lined left">Effortless Bitcoin Purchases</h2>
+
+                                <p>With the Wasabi Wallet "Buy Anything with Bitcoin" feature, you can purchase flights, cars, or even ASIC miners with ease, all while maintaining your privacy and from the comfort of your wallet.</p>
+                                <a href="index.html#download"><img src="img/bab.svg" class="w-50 my-3" />
+                                </a>
+                            </div>
+                        </div>
+                        <div class="col-md-6 offset-md-1">
+                            <div class="row my-5">
+                                <div class="col-12 mb-4">
+                                    <div class="feature-box">
+                                        <div class="title my-3 green-text">1. Your Personal Concierge</div>
+                                        <p>Our feature connects you with a personal concierge from ShopinBit, ensuring a private and secure shopping experience.</p>
+                                    </div>
+                                </div>
+                                <div class="col-12 mb-4">
+                                    <div class="feature-box">
+                                        <div class="title my-3 green-text">2. Free Shipping & More</div>
+                                        <p>Enjoy free shipping within the EU, access millions of products, and uncover incredible travel deals while Buying Anything with Bitcoin on Wasabi Wallet.</p>
+                                    </div>
+                                </div>
+                                <div class="col-12 mb-4">
+                                    <div class="feature-box">
+                                        <div class="title my-3 green-text">3. Privacy-First Shopping</div>
+                                        <p>Wasabi Wallet and ShopinBit prioritize your privacy, sharing only the necessary information for order fulfillment.</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+
+                </div>
+            </section>
+            <section class="home p-4 green-bg">
+                <div class="container mb-5">
+                    <div class="row align-items-center">
+                        <div class="col-12 col-md-6">
+                            <div class="my-5">
+                                <h2 class="pt-header title-lined left">Exclusive Rewards</h2>
+                                <p class="big-text">Don't miss out on our exclusive offer! Random buyers using the new feature stand a chance to win <strong>$100 vouchers</strong>. Act fast, offer ends December 14th XX:XX UTC.</p>
```

Comment:
This isn't ever green content, remember to remove it later.

Reply:
date is already over. remove or update it

Reply:
Reminder that this needs to be changed or removed.